sentry-android-core/src/main/java/io/sentry/android/core/ActivityLifecycleIntegration.java

```diff
@@ -99,6 +100,10 @@ public void register(final @NotNull IHub hub, final @NotNull SentryOptions optio
     if (this.options.isEnableActivityLifecycleBreadcrumbs() || performanceEnabled) {
       application.registerActivityLifecycleCallbacks(this);
       this.options.getLogger().log(SentryLevel.DEBUG, "ActivityLifecycleIntegration installed.");
+      final SdkVersion sdkVersion = this.options.getSdkVersion();
+      if (sdkVersion != null) {
+        sdkVersion.addIntegration("Activity");
```

Comment:
`l`: Maybe we should keep `Lifecycle`? We have `AppLifecycle` below.
```suggestion
        sdkVersion.addIntegration("ActivityLifecycle");
```